### PR TITLE
Link V2 project payout with handle properly

### PR DESCRIPTION
**Pull request by tomquirk**
Make splits to projects that have a handle link to their handle.

A smol cleanup otherwise.

### CODE REVIEW

File: src/components/v2/shared/SplitItem.tsx [hunk 1]
```diff
@@ -1,25 +1,37 @@
...
 import { Split } from 'models/v2/splits'
 import Link from 'next/link'
+import { useContext } from 'react'
+import { formatDate } from 'utils/formatDate'
 import { formatWad } from 'utils/formatNumber'
 import { v2ProjectRoute } from 'utils/routes'
 import { V2CurrencyName } from 'utils/v2/currency'
 import { formatSplitPercent, SPLITS_TOTAL_PERCENT } from 'utils/v2/math'
 
+const LockedText = ({ lockedUntil }: { lockedUntil: number }) => {
```
Comment (tomquirk) [inline src/components/v2/shared/SplitItem.tsx:22]: Move out of scope cos it doesn't need to be in scope.

File: src/components/v2/shared/SplitItem.tsx [hunk 2]
```diff
@@ -81,24 +76,21 @@ export default function SplitItem({
       }
       return t`Unknown Project`
     }
+
     return (
       <div>
         <div>
-          <Tooltip title={getProjectTooltip()}>
```
Comment (tomquirk) [inline src/components/v2/shared/SplitItem.tsx:87]: Kill the tooltip. KILL THE TOOLTIP

File: src/components/v2/shared/SplitItem.tsx [hunk 3]
```diff
@@ -81,24 +76,21 @@ export default function SplitItem({
...
-                  className="text-primary hover-text-action-primary hover-text-decoration-underline"
-                  style={{ fontWeight: 500 }}
-                >
-                  {getProjectLabel()}
-                </a>
-              </Link>
-            </span>
-          </Tooltip>
+          <Link
+            href={v2ProjectRoute({ projectId: split.projectId, handle })}
```
Comment (tomquirk) [inline src/components/v2/shared/SplitItem.tsx:84]: this line is the actual bug fix

**Review by tomquirk — COMMENTED** [review]
@wraeth-eth can you pls review when you have a chance ❤️

File: src/components/v2/shared/SplitItem.tsx [hunk 4]
```diff
@@ -81,24 +76,21 @@ export default function SplitItem({
...
+            target="_blank"
```
Comment (wraeth-eth) [inline src/components/v2/shared/SplitItem.tsx:85]: The redirect isn't working properly - I think maybe this needs to be on the a tag itself

**Review by wraeth-eth — CHANGES_REQUESTED** [review]
.

File: src/components/v2/shared/SplitItem.tsx (hunk 4 above)
Comment (tomquirk) [reply src/components/v2/shared/SplitItem.tsx]: Ahh great great catch.

Fixed

**Comment by tomquirk** [comment]
friendly bump @wraeth-eth

**Review by wraeth-eth — APPROVED** [review]

**Pull request merged**